class-tgm-plugin-activation.php

```diff
@@ -1519,7 +1519,9 @@ protected function get_plugins_api( $slug ) {
 
 				if ( is_wp_error( $response ) ) {
 					if ( true === WP_DEBUG ) {
+						// @codingStandardsIgnoreStart
 						wp_die( esc_html( $this->strings['oops'] ) . var_dump( $api ) ); // WPCS: xss ok.
+						// @codingStandardsIgnoreEnd
 					} else {
```

Comment:
+1 on dropping of the `var_dump()`

Re - local runs - I run with a stricter/adjusted ruleset locally (as I presume you do), so that only applies for other contributors